.github/workflows/ios-release.yml

```diff
@@ -18,6 +18,10 @@ jobs:
         fetch-depth: 0  # Ensure that we can operate on the full history
         ref: main
 
+    - uses: maxim-lobanov/setup-xcode@v1
+      with:
+        xcode-version: '16.0'
```

Comment:
Had to bump this to resolve some whack build error that I couldn't repro locally. Can bump to 16.1 whenever GitHub updates (final release just dropped).